examples/react-apollo-crm/package.json

```diff
@@ -1,15 +1,15 @@
 {
-  "name": "react-apollo",
+  "name": "@nhost-examples/react-apollo",
   "version": "0.1.0",
   "private": true,
   "dependencies": {
     "@apollo/client": "^3.4.16",
     "@craco/craco": "^6.4.0",
     "@headlessui/react": "^1.4.2",
     "@heroicons/react": "^1.0.5",
-    "@nhost/nhost-js": "^0.3.4",
-    "@nhost/react-apollo": "^2.0.7-0",
-    "@nhost/react-auth": "^2.0.3",
+    "@nhost/nhost-js": "workspace:*",
+    "@nhost/react-apollo": "workspace:*",
+    "@nhost/react-auth": "workspace:*",
```

Comment:
How will this work for developers who copy the `react-apollo-crm` folder to a separate folder and runs this example independently?

Reply:
corrected. this syntax is only useful to generate version range when publishing packages to npm. Examples are not published as packages therefore don't need this.